File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_46959.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest46959():
    field_value = request.form.get('field', '')
    data = (lambda v: v.strip())(field_value)
    if not str(data).isdigit():
        raise Exception('error: ' + str(data))
    return jsonify({"result": "success"})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest46959.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest46959 {

    @GetMapping("/BenchmarkTest46959")
    public void BenchmarkTest46959(@RequestHeader("Authorization") String authorization, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String authHeader = authorization != null ? authorization : "";
        request.getSession().setAttribute("data", String.valueOf(authHeader));
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
